IDEAS/Buildings/Components/Interfaces/PartialZone.mo

```diff
@@ -3,6 +3,10 @@ partial model PartialZone "Partial model for thermal building zones"
   replaceable package Medium =
     Modelica.Media.Interfaces.PartialMedium "Medium in the component"
       annotation (choicesAllMatching = true);
+
+  parameter Modelica.Fluid.Types.Dynamics energyDynamicsAir=Modelica.Fluid.Types.Dynamics.FixedInitial
+    "Type of energy balance for air model: dynamic (3 initialization options) or steady state";
+  parameter Real mSenFac = 5 "Correction factor for thermal capacity of zone air.";
```

Comment:
The factor 5 has nothing to do with furniture, that's the story everyone tells who doesn't know the old literature ;) The factor is included to translate local phenomenon to a global phenomenon because you only use 1 air temperature (while you can imagina e.g. the aire temperature next to a radiator is different than the global temperature). Measurements show 5 is a good approximation for (semi-)convective heat emission systems.

PS. If you also want to include the effect of furniture, it should be around 10.